package.json

```diff
@@ -42,6 +42,7 @@
     "@angular/router": "^8.2.14",
     "@angular/service-worker": "^8.2.13",
     "@nestjs/common": "^6.8.3",
+    "@nestjs/config": "^0.4.2",
```

Comment:
Isn't used anywhere in the project.

Should we more until we need it?

Reply:
This seems to have been removed already, idk when

Reply:
Nope, I'm a moron; was looking in the idp cherry branch when I looked for this.